### DISCUSSION TITLE
Single VFO?

### DISCUSSION THREAD

**Discussion by oisact** [egzumer/uv-k5-firmware-custom#217] [Q&A]
Is there a way to disable the dual-VFO mode (especially since this radio doesn't actually have two VFOs)?

**Comment by egzumer** [comment]
Yes, by switching RXMode, read the manual

**Reply by oisact** [reply]
No, that's not what I'm talking about. That merely controls the pseudo-dual-vfo behavior, which other radios call "dual watch". What I'm talking about is the true single-VFO mode, supported by everything from AnyTone to Yeasu HTs. When in single-VFO mode the entire display is dedicated to displaying only the selected VFO A or B. The font for the frequency / memory channel name is larger, etc, and more space is available for showing detailed information about that single VFO.

**Reply by egzumer** [reply]
`MAIN ONLY` is functionally the real single VFO mode. 

A single VFO display on the other hand is not on my priority list right now. It would be somewhat more clean, but it won't be such a big improvement. The font won't be any bigger, it won't contain any more information than there already is, but it will certainly eat up more memory space though.

**Comment by f5gkw** [comment]
This feature exists on IJV firm.

**Comment by ErikS-web** [comment]
See also issue #124. 

There's no need to make a/extra bigger font.
